tests/everflow/templates/acl_rule_v6.json.j2

```diff
@@ -0,0 +1,474 @@
+                                    "config": {
+                                        "source-ip-address": "ffbe:0225:7c6b:a982:d48b:230e:f271:000b/128",
+                                        "destination-ip-address": "ffbe:0225:7c6b:a982:d48b:230e:f271:000a/128",
+                                        "protocol": 17,
+                                        "dscp": 8
+                                    }
+                                },
+                                "transport": {
+                                    "config": {
+                                        "source-port": "514",
+                                        "destination-port": "12001"
+                                    }
+                                }
+                            },
+                            "18": {
+                                "actions": {
+                                    "config": {
+                                        "forwarding-action": "ACCEPT"
+                                    }
+                                },
+                                "config": {
+                                    "sequence-id": 18
+                                },
+                                "ip": {
+                                    "config": {
+                                        "source-ip-address": "ffbe:0225:7c6b:a982:d48b:230e:f271:000c/128",
+                                        "destination-ip-address": "ffbe:0225:7c6b:a982:d48b:230e:f271:000d/128"
+                                    }
+                                },
+                                "transport": {
+                                    "config": {
+                                        "source-port": "12002",
+                                        "destination-port": "12003"
+                                    }
+                                }
+                            },
+                            "19": {
+                                "actions": {
+                                    "config": {
+                                        "forwarding-action": "ACCEPT"
+                                    }
+                                },
+                                "config": {
+                                    "sequence-id": 19
+                                },
+                                "ip": {
+                                    "config": {
+                                        "source-ip-address": "ffbe:0225:7c6b:a982:d48b:230e:f271:000e/128",
+                                        "destination-ip-address": "ffbe:0225:7c6b:a982:d48b:230e:f271:000f/128",
+                                        "protocol": 17,
+                                        "dscp": 16
+                                    }
+                                },
+                                "transport": {
+                                    "config": {
+                                        "source-port": "12004",
```

Comment:
is this negative test case ? Protocol being UDP but match on TCP Flags. What is expected behaviour here ?

Reply:
We want to see if the ASIC crashes when we install this rule. The idea being that the user could specify an arbitrary protocol number and we want to make sure that we don't crash when the protocol number conflicts with other fields in the ACL rule.

Reply:
ok, then will Sonic return error and there will be syslog message for same ?

Reply:
Nope, no error should occur. The ASIC should be able to handle arbitrary protocol numbers supplied by the user. I think it might be good to include some other random protocol numbers as well just to be sure.